resolwe_bio/tests/files/wgs/input/TP53.bed

```diff
@@ -0,0 +1 @@
+17	7578063	7578234	TP53_31_59.chr17.7578190.7578536_tile_1
```

Comment:
Some of these testing files are already present in `./bqsr` folder. What was the logic behind duplicating them?